k8s/config.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+clusters:
+  - cluster:
+      certificate-authority-data: KUBE_CLUSTER_CERTIFICATE
```

Comment:
Maybe use $ as prefix to make it clear to anyone finding just the config.yaml that it's meant to be replaced. Example: $KUBE_USER_NAME